admin-dev/ajax-tab.php

```diff
@@ -23,11 +23,8 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-
-umask(0000); // This will let the permissions be 0777
-
 if (!defined('_PS_ADMIN_DIR_')) {
-    define('_PS_ADMIN_DIR_', getcwd());
+    define('_PS_ADMIN_DIR_', __DIR__);
```

Comment:
This seems to be out of the scope of this PR ?

Reply:
Yes, but it's a bug :/
I forgot to update the description ^^

Reply:
I'm worried about the impact of this change for some server configuration. Like shops in sub-folders.
Shall we require extensive QA tests ?

Reply:
Don't know how it can have an impact, there is a huge difference between cwd & dir.
If you are running the script in other directory than the admin, it failed :)

Reply:
Agreed with @PierreRambaud `getcwd` return the execution path not the file path, which could have really bad side effects in this case. However in our usual web context where they end up having the same value it is safe to have this modification.
I don't think a thorough QA needs to be done, simply check each modified end point, if something is broken it should break instantly anyway.